include/deal.II/base/tensor.h

```diff
@@ -2613,6 +2617,50 @@ cofactor(const Tensor<2, dim, Number> &t)
 }
 
 
+/**
+ * Return the nearest orthogonal matrix using a SVD if the determinant is
+ * more than a tolerance away from one. The orthogonalization is done by
+ * combining the products of the SVD decomposition: $((U V^T)^T$, where
```

Comment:
I think I haven't quite understand the purpose of the last `^T` yet. Apologies for not picking up on that earlier.

Here's what I think you want: Let `A` be a matrix. if `det A \approx 1`, then the matrix is already orthogonal and you just return `A`. If it isn't already orthogonal, then you do `A = U S V^T` and you want to return `B = U V^T`. (This is a projection because if you apply the operation twice, you're going to get `B` again.) But the documentation says that you're not going to return `B = U V^T`, but instead `B^T = (U V^T)^T`. I don't understand the transposition at the end -- to make that compatible with the case where the determinant is one, you would have to return `A^T` in the first case as well, but you don't.

Reply:
```suggestion
 * combining the products of the SVD decomposition: $(U V^T)^T = V U^T$, where
```

Reply:
Hey Wolfgang,

You are right, the transposed in the documentation is incorrect. The correct equation is I believe to remove the outer transposed from it, and I have done so.

It might be good explain how it ended up there in the first place. While making this function I had to deal with the fact that the LapackFullMatrix class stores the matrix in a transposed way. So I had to re transpose the answer. To remind myself of that I had written it down the way I did. 

Later I changed to code to use the copy functions, which automatically account for that (see https://dealii.org/developer/doxygen/deal.II/classLAPACKFullMatrix.html#a0a7678e72a99b1adbfcf72c76bbc66e8), but when expanding the documentation, I did not update that part of the equation. I think that it is correct now.